hw/usb/tinyusb/da146xx/src/da1469x.c

```diff
@@ -66,9 +90,59 @@ tinyusb_hardware_init(void)
     if (MYNEWT_VAL_CHOICE(DA1469X_USB_VBUS_HANDLING, auto)) {
         /* If VBUS handling is set to 'auto' let da1469x_vbus module handle notification */
         da1469x_vbus_add_handler(tinyusb_vbus_changed);
-    }
-    else if (MYNEWT_VAL_CHOICE(DA1469X_USB_VBUS_HANDLING, ignore)) {
+    } else if (MYNEWT_VAL_CHOICE(DA1469X_USB_VBUS_HANDLING, ignore) ||
+               MYNEWT_VAL_CHOICE(DA1469X_USB_VBUS_HANDLING, custom)) {
         /* If VBUS handling is ignored notify TinyUSB that VBUS is on */
-        tinyusb_vbus_changed(true);
+#if MYNEWT_VAL(DA146XX_USB_MONITOR)
+        /* With USB Monitor enabled, don't immediately notify VBUS on
+         * Wait for actual USB activity (Setup packet) to be detected
+         * by USB Monitor before notifying TinyUSB of VBUS presence.
```

Comment:
This comment looks strange.

VBUS notification is normally used to connect USB device to  host by connection pull up resistor.
If application for some reason ignores VBUS (present of not) and makes call to tusb_connect(0) to start activity on USB, I don't see any reason to later notify USB about VBUS that will call tusb_connect again.
First call will start USB activity regardless of VBUS.
VBUS is not used for anything else apart from initiating connection and disconnecting when it's gone.